docs/hyperopt.md

```diff
@@ -303,8 +303,9 @@ Given the following result from hyperopt:
 
 ```
 Best result:
-   135 trades. Avg profit  0.57%. Total profit  0.03871918 BTC (0.7722Σ%). Avg duration 180.4 mins.
+    44/100:    135 trades. Avg profit  0.57%. Total profit  0.03871918 BTC (0.7722Σ%). Avg duration 180.4 mins. Objective: 1.94367
 with values:
```

Comment:
Yes, please remove this line - it's pointless considering the next line.

Reply:
Done. I added empty lines before and after best result string for better readability. Docs also adjusted with updated format of the output.